js/chat_app/src/modals/ChatForm/Select.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import TextField from '@mui/material/TextField';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import { useDispatch, useSelector } from 'react-redux';
+import { setSelectedMembers } from '../../redux/slices/chatSlice';
+
+export default function CreateChatSelect({ resetMembersTrigger }) {
+  const [membersName, setMembersName] = useState([]);
+  const authUsername = useSelector((state) => state.auth.auth.user.username);
+  const users = useSelector((state) => state.users.allUsers);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    setMembersName([]);
+  }, [resetMembersTrigger]);
+
+  const handleChange = (event) => {
+    const { target: { value } } = event;
+
+    setMembersName(
+      // On autofill we get a stringified value.
+      typeof value === 'string' ? value.split(',') : value
+    );
+
+    const membersIds = users.filter((user) => value.includes(user.username)).map((user) => user.id);
+    dispatch(setSelectedMembers(membersIds));
+  };
+
+  return (
+    <div>
+      <FormControl
+        fullWidth
+        sx={{
+          width: '350px'
+        }}
+      >
+        <TextField
+          select
+          label="Members"
+          variant="outlined"
+          SelectProps={{
+            multiple: true,
+            onChange: handleChange,
+            value: membersName
+          }}
+        >
+          {users.map((user) => {
+            if (user.username !== authUsername) {
```

Comment:
use ternary operators instead. example
`user.username !== authUsername ? <COMPONENT> : null`

Reply:
eslint has error when i do it, if it`s importatnt i can try to fix this